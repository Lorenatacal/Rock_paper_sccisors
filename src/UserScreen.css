.split {
    height: 80%;
    margin-bottom: 2%;
    width: 50%;
    position: absolute;
    z-index: 1;
    overflow-x: hidden;
    padding-top: 20px;
}
.left {
    left: 2%;
    right: 1%;
    background-color: rgb(228, 227, 227);
    color: black;
  }