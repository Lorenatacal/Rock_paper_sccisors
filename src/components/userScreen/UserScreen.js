import React, { Component } from "react";
import Options from "../options/Options";
import AddName from "../addName/AddName";
import styled from "styled-components";

const StyledUser = styled.div`
  height: 53%;
  margin-top: 1%;
  margin-bottom: 1%;
  width: 47%;
  position: absolute;
  z-index: 1;
  overflow-x: hidden;
  padding-top: 20px;
  right: 2%;
  background-color: rgb(241, 239, 239);
  left: 2%;
  right: 1%;
  color: black;
`;

class UserScreen extends Component {
  render() {
    return (
      <StyledUser>
        <AddName />
        <Options action={this.props.action} />
      </StyledUser>
    );
  }
}
export default UserScreen;
