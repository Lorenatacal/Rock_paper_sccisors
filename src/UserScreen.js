import React, { Component } from 'react';
import './UserScreen.css';
import AddName from './AddName';

class UserScreen extends Component {
    render() {
        return (
            <div class= "split left">
                <AddName />
            </div>
        );
    }
}
export default UserScreen;