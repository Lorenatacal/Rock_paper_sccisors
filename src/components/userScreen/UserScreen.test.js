import React, { Component } from "react";
import UserScreen from "./UserScreen";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

Enzyme.configure({ adapter: new Adapter() });

test("UserScreen should render correctly", () => {
  const wrapper = Enzyme.shallow(<UserScreen />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
