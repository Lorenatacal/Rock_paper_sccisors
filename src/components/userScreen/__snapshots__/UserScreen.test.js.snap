// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserScreen should render correctly 1`] = `
<styled.div>
  <AddName />
  <styled.span>
    Please select your respone
  </styled.span>
  <Options />
</styled.div>
`;
